### PR TITLE
Feature/5641 add default checks

### PR DESCRIPTION
**Pull request by zorafuchs**

### CODE REVIEW

File: lib/renuo_bin_check/default_scripts/default_scripts.rb [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+  def console_log
+    @default_scripts << DSLConfig.new('console_log') do
+      command "grep -i -r 'console.log' app spec"
+      reversed_exit true
+      files ['app/**/*', 'spec/**/*']
+      success_message '+console.log found. Please fix them and try again, commit aborted'
+    end
+  end
+
+  def put_without_brackets
```
Comment (schmijos) [inline lib/renuo_bin_check/default_scripts/default_scripts.rb]: Please rename to `puts_without_brackets`

File: lib/renuo_bin_check/default_scripts/default_scripts.rb [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+    @default_scripts << DSLConfig.new('put_without_brackets') do
+      command "grep -i -r '  puts ' app spec"
+      reversed_exit true
+      files ['app/**/*', 'spec/**/*']
+      success_message '+puts found. Please fix them and try again, commit aborted'
+    end
+  end
+
+  def put_with_brackets
```
Comment (schmijos) [inline lib/renuo_bin_check/default_scripts/default_scripts.rb]: Please rename to `puts_without_brackets`

File: lib/renuo_bin_check/default_scripts/default_scripts.rb [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+    @default_scripts << DSLConfig.new('put_with_brackets') do
```
Comment (schmijos) [inline lib/renuo_bin_check/default_scripts/default_scripts.rb]: Please rename to `puts_without_brackets`

File: lib/renuo_bin_check/dsl_config.rb [hunk 4]
```diff
@@ -15,7 +15,7 @@ def method_missing(name, *_params, &configs)
     end
   end
 
-  def respond_to_missing?
+  def respond_to_missing?(_method_name, _include_private)
     #:nocov:
     true
     #:nocov:
```
Comment (schmijos) [inline lib/renuo_bin_check/dsl_config.rb]: Why not covering that with a test?

Comment (zorafuchs) [inline lib/renuo_bin_check/dsl_config.rb]: robocop wants it
we tried to test it, but somehow it never worked...

**Pull request merged**